web/src/components/RemoteContent.tsx

```diff
@@ -61,6 +61,13 @@ const RemoteContent = ({ html, centered = false, smallText = false }: RemoteCont
       return
     }
     const currentSandBoxRef = sandBoxRef.current
+
+    const allElements = currentSandBoxRef.querySelectorAll('*')
+    allElements.forEach(el => {
+      const element = el as HTMLElement
+      element.style.removeProperty('color')
```

Comment:
🤔 What do you think of this solution? This avoids removing all colors which might be quite a big change for some regions.
```suggestion
      if (element instanceof HTMLElement && element.style.color === 'rgb(0, 0, 0)') {
        element.style.removeProperty('color')
      }
```

Reply:
maybe we could change it only when it is contrast theme. Otherwise there might be some colors which might have less contrast and totally get blended with the background... :thinking:

Reply:
in the case above there is no check for contrast theme, so it will be applied for both of the themes. But I can implement your suggestion with checking theme type and it is better solution :+1:

Reply:
Hmm, not sure if the check for the theme is actually necessary if we only do it for black anyway? In the case of the light theme there should not be a difference if we remove black colors.